packages/cyberstorm/src/components/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { MouseEventHandler } from "react";
+import styles from "./componentStyles/Button.module.css";
+
+export interface ButtonProps {
+  label?: string;
+  leftIcon?: string;
```

Comment:
The icons are still typed as strings.

Reply:
changed the type to Element. Is Element good, or should it be something else?

Reply:
I think we might want to go with `ReactNode`.

Reply:
Check